examples/raylib/raylib.c

```diff
@@ -46,27 +48,31 @@ void raylib_free_image(void *ptr)
     free(ptr);
 }
 
-void draw_polyline(double **points, double x, double y, int pointsc)
+void draw_polyline(double **points, double x, double y, int pointsc, struct Color color)
```

Comment:
Now these custom functions from SDL example use Color as all Raylib functions.
No global Color in this example.